### ISSUE TITLE
MediaLibrary photos are blank on Testflight

### ISSUE THREAD

**Issue by Dagurmart** [expo/expo#6524]
## 🐛 Bug Report

Hi. I'm displaying a list of photos from the camera roll using MediaLibrary.getAssetsAsync so the user can select multiple images. They appear on Android and on iOS if I'm running it through expo. However, if I build the app and upload it to Testflight, I get the list of images but they're all blank. I can still select them and the selected images appear just fine on the next screen. 
There are no issues on Android, expo or play store.

Edit: I should add that on Expo 35 we were using CameraRoll.getPhotos(). That stopped working after the Expo 36 upgrade so we switched to MediaLibrary.getAssetsAsync.

### Environment

  Expo CLI 3.11.1 environment info:
    System:
      OS: Linux 5.3 Ubuntu 19.10 (Eoan Ermine)
      Shell: 5.0.3 - /bin/bash
    Binaries:
      Node: 10.15.2 - /usr/bin/node
      Yarn: 1.19.2 - /usr/bin/yarn
      npm: 6.4.1 - /usr/local/bin/npm
    npmPackages:
      expo: ^36.0.0 => 36.0.0 
      react: 16.9.0 => 16.9.0 
      react-native: 0.61.4 => 0.61.4 
      react-navigation: ^4.0.7 => 4.0.10 
    npmGlobalPackages:
      expo-cli: 3.11.1

Targets: Android and iOS

### Steps to Reproduce

I get the photos like this
```
      const album = await MediaLibrary.getAlbumAsync('Camera')
      const photosTemp = await MediaLibrary.getAssetsAsync({ album: album })
      const array = photosTemp.assets.map(asset => ({
        ...asset,
        type: asset.mediaType,
        timestamp: asset.creationTime,
        selected: false
      }))
```
and display them in a FlatList

```
<FlatList
                keyExtractor={(item, index) => item.uri}
                numColumns={numColumns}
                data={cameraRoll}
                renderItem={({ item, index }) => renderItem(item, index)}
                style={style.container}
                getItemLayout={getItemLayout}
                onEndReached={getPhotos}
                contentContainerStyle={{ paddingBottom: 80 }}
              />
```

Where renderItem is
```
<TouchableOpacity>    
    <Image style={style.listItem} resizeMode={'cover'} source={{ uri: item.uri }} />
</TouchableOpacity>
```

### Expected Behavior

I expected the same behavior on Expo and Testflight

### Actual Behavior

Images are blank on Testflight

### Reproducible Demo

Not sure if I can do this without paying Apple

**Comment by cruzach** [comment]
Hm that's odd, sorry you're having so much trouble with this. `CameraRoll` was dropped as part of the `react-native` upgrade, but we could've communicated that a little better.

Can you make sure you have `Permissions.CAMERA_ROLL` before you're requesting? The fact that you're able to view the images on the next screen makes me think this might not be a bug with Expo, but I could be wrong. It would help if you could just provide code that reproduces this, and I can submit a testflight build on my own

**Comment by rxhivzero** [comment]
same problem

**Comment by rxhivzero** [comment]
Maybe this problem?

https://github.com/expo/expo/pull/6344#issuecomment-564909015

**Comment by rxhivzero** [comment]
My expokit add @react-native-community/cameraroll can work,
but the same problem with expo build.

**Comment by Dagurmart** [comment]
> Hm that's odd, sorry you're having so much trouble with this. `CameraRoll` was dropped as part of the `react-native` upgrade, but we could've communicated that a little better.
> 
> Can you make sure you have `Permissions.CAMERA_ROLL` before you're requesting? The fact that you're able to view the images on the next screen makes me think this might not be a bug with Expo, but I could be wrong. It would help if you could just provide code that reproduces this, and I can submit a testflight build on my own

I also suspected that it could be a permission issue but I made sure that it wasn't the case. CAMERA_ROLL was permitted

**Comment by tsapeta** [comment]
Hi @Dagurmart,
Yeah, this issue sounds a bit related to #6344, really sorry for the trouble 🙇
I'll take care of this issue and will keep you updated!

**Comment by tsapeta** [comment]
The fix for this has landed on our fork of `react-native` in `sdk-36.0.1` tag and it can be used in ExpoKit projects by changing the source of `react-native` dependency in your `package.json` file to
`https://github.com/expo/react-native/archive/sdk-36.0.1.tar.gz` and running `yarn` or `npm install` and then `pod install` in `ios` directory 🙂 

Now I'm going to roll out this fix to standalone app builders - I'll let you know once it's out 😉

**Comment by tsapeta** [comment]
This fix has been successfully deployed to our standalone builders - so, you can just try out building a new binary 😉 Sorry it took so long but we've had one small issue to resolve that has been blocking us from deploying it, but now everything is ready to go! 😊
